app/src/main/java/com/spendster/presentation/address/AddressActivity.java

```diff
@@ -1,15 +1,61 @@
 package com.spendster.presentation.address;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
 
 import com.spendster.R;
 
-public class AddressActivity extends AppCompatActivity {
+public class AddressActivity extends AppCompatActivity implements AddressView {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_adress);
+
+        final EditText etFLName = findViewById(R.id.etFLName);
+        final EditText etPhone = findViewById(R.id.etPhone);
+        final EditText etAddress = findViewById(R.id.etAddress);
+
+        final Button btnGetStarted = findViewById(R.id.btnGetStarted);
+
+        final AddressPresenter addressPresenter = new AddressPresenter(this);
+
+        btnGetStarted.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                String fLName = etFLName.getText().toString();
```

Comment:
@SimonMaha these lines should be extracted into functions as well. Also, don't forget to trim text before getting it from text fields.